proto/src/determined/command/v1/command.proto

```diff
@@ -4,11 +4,24 @@ package determined.command.v1;
 option go_package = "github.com/determined-ai/determined/proto/pkg/commandv1";
 
 import "google/protobuf/timestamp.proto";
+import "protoc-gen-swagger/options/annotations.proto";
 
 import "determined/container/v1/container.proto";
 
 // Command is a single container running the configured command.
 message Command {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: {
+      required: [
+        "id",
+        "description",
+        "start_time",
+        "container",
```

Comment:
container can be missing.

Reply:
same for other 'container' attributes in the other commands

Reply:
That means status might be unavailable since it's a child of container.

Is it ok to default the state to PENDING if unavailable?

Reply:
yep. that's that it should be. I added more details in a comment below

Reply:
Thanks, can you please check the changes to:
- proto/src/determined/api/v1/tensorboard.proto
- proto/src/determined/container/v1/container.proto